configure.py

```diff
@@ -1899,7 +1905,9 @@ def query_seastar_flags(pc_file, link_static_cxx=False):
                 f.write('build $builddir/{}/{}_g: {}.{} {} | {} {}\n'.format(mode, binary, regular_link_rule, mode, str.join(' ', objs), seastar_dep, seastar_testing_dep))
                 f.write('   libs = {}\n'.format(local_libs))
             else:
-                f.write('build $builddir/{}/{}: {}.{} {} | {}\n'.format(mode, binary, regular_link_rule, mode, str.join(' ', objs), seastar_dep))
+                if binary == 'scylla':
+                    local_libs += ' ' + "$seastar_testing_libs_{}".format(mode)
```

Comment:
Not thrilled about adding all this test stuff to scylla.
Maybe we should look into loosing the test dependency first.

Reply:
Might not be worth it. If it is too complicated, we can defer it to a follow-up.

Reply:
we are using `src/testing/random.cc` which is compiled in `seastar_testing`.

Reply:
> Not thrilled about adding all this test stuff to scylla.
> Maybe we should look into loosing the test dependency first.

i did look into this. no luck.

Reply:
i added more details in the commit message. HTH.